spec/helpers/util.js

```diff
@@ -60,6 +60,14 @@ function isIE() {
     return ((navigator.appName === 'Microsoft Internet Explorer') || ((navigator.appName === 'Netscape') && (new RegExp('Trident/.*rv:([0-9]{1,}[.0-9]{0,})').exec(navigator.userAgent) !== null)));
 }
 
+function getEdgeVersion() {
+    var match = /Edge\/(\d+)/.exec(navigator.userAgent);
+    if (match !== null) {
+        return new Number(match[1]);
```

Comment:
This was an oldschool js trick I used to use to convert a decimal number within a string into a primitive number (ie `'12.123'` into `12.123`). Definitely open to a better way to do this.